aggregator/aggregator.go

```diff
@@ -325,7 +326,7 @@ func (agg *Aggregator) Close() error {
 func (agg *Aggregator) sendAggregatedResponseToContract(blsAggServiceResp blsagg.BlsAggregationServiceResponse) {
 	if blsAggServiceResp.Err != nil {
 		agg.aggregatorListener.IncErroredSubmissions()
-		if errors.Is(blsAggServiceResp.Err, blsagg.TaskExpiredError) {
+		if strings.Contains(blsAggServiceResp.Err.Error(), "expired") {
```

Comment:
Is there no support for `errors.Is` for this particular one?

Reply:
It's now an error function, so it's never the same error. This is honestly such a bummer, error handling should be easier in go. Not sure if there's a better way to do that, but seems unlikely